packages/math-input/src/components/empty-keypad-button.tsx

```diff
@@ -42,7 +44,7 @@ class EmptyKeypadButton extends React.Component<Props> {
     }
 }
 
-const mapStateToProps = (state) => {
+const mapStateToProps: (state: State) => ReduxProps = (state) => {
```

Comment:
This level of type annotation isn't necessary.  The following should be enough:
```
const mapStateToProps = (state:State): ReduxProps => {
```
If that works, could you update the rest of the `mapStateToProps` definitions?